internal/connector/internal/workers/connector_mgr.go

```diff
@@ -50,13 +50,15 @@ func NewConnectorManager(
 	vaultService vault.VaultService,
 	bus signalbus.SignalBus,
 	db *db.ConnectionFactory,
+	reconcilerConfig *workers.ReconcilerConfig,
 ) *ConnectorManager {
 	result := &ConnectorManager{
 		BaseWorker: workers.BaseWorker{
 			Id:         uuid.New().String(),
 			WorkerType: "connector",
 			Reconciler: workers.Reconciler{
-				SignalBus: bus,
+				SignalBus:        bus,
+				ReconcilerConfig: reconcilerConfig,
```

Comment:
@valdar @dhirajsb We'd like to get your review on this change from the connectors side.
Thanks